OmsiHook/WrappedOmsiClasses/OmsiComplMapObjInst.cs

```diff
@@ -205,6 +213,48 @@ public void SetStringVariable(string varName, string value)
             stringVars[index] = new(value);
         }
 
+        /// <summary>
+        /// Get a constant for an object from its name.
+        /// </summary>
+        /// <param name="varName">Const Name</param>
+        /// <returns>requested float value</returns>
+        /// <exception cref="KeyNotFoundException"/>
+        public float GetConst(string varName)
+        {
+            int index = constStrings[varName];
+            if (index >= consts.Length || index < 0)
+                throw new KeyNotFoundException($"Const Variable '{varName}' not found in object. - Index Out Of Bounds");
+            return consts[index];
+        }
+
+        public float GetCurve(string varName, float x)
+        {
+            int index = funcsStrings[varName];
+            if (index >= consts.Length || index < 0)
+                throw new KeyNotFoundException($"Curve Variable '{varName}' not found in object. - Index Out Of Bounds");
+
+            for (int i = 0; i < funcs[index].Pnts.Length - 1; i++)
+            {
+                float x1 = funcs[index].Pnts[i].x;
```

Comment:
You really don't want to be accessing `OmsiFuncClass.Pnts` every iteration as it calls the getter which reads a new copy of the entire array of points from memory every time it's accessed.